frontend/src/common/components/formInputField/FormNumberInputField.tsx

```diff
@@ -30,12 +30,21 @@ export default function FormNumberInputField({
     const bannedCharacters = ["e", "E", "+", "-"];
     if (fractionDigits === 0) bannedCharacters.push(".");
 
+    const handleWheel = (e) => {
+        e.target.blur();
+        e.stopPropagation();
+        setTimeout(() => {
+            e.target.focus();
+        }, 0);
+    };
```

Comment:
Even with this the "input is focused/unfocused" animation is spammed on the element when scrolling very slowly, but still, this simply seems like the best way to disable scrolling on the NumberInput.

Something I would add to this however, is checking if the current element is focused before running the function (`if (document.activeElement === e.target) {...}`).
This will stop changing the focus to a different number input when scrolling (and there is no need to run the function if user is scrolling and cursor is. over an unfocused input)

Reply:
Good observation! Added!